src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
+    
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int FIRST_ELEMENT = 0;
+    private static final int SECOND_ELEMENT = 1;
```

Comment:
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int AMOUT_INDEX = 1;
```
https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#remember-about-informative-name-of-the-variables